addons/godot-firebase/firestore/firestore.gd

```diff
@@ -369,6 +369,7 @@ func _on_FirebaseAuth_token_refresh_succeeded(auth_result : Dictionary) -> void:
 
 
 func _on_pooled_request_completed(result : int, response_code : int, headers : PoolStringArray, body : PoolByteArray, request : HTTPRequest) -> void:
+    print(JSON.parse(body.get_string_from_utf8()).result)
```

Comment:
Unless absolutely needed by the user, this should probably be removed.

Reply:
Looks like this piece was already there (ore some commits ago?) since I didn't add it. Will remove it soon